app/src/main/java/com/github/fribourgsdp/radio/LocalDatabase.kt

```diff
@@ -0,0 +1,44 @@
+package com.github.fribourgsdp.radio
+
+import com.google.android.gms.tasks.Task
+import com.google.android.gms.tasks.Tasks
+import java.lang.IllegalArgumentException
+
+class LocalDatabase : Database {
+    private val userMap: MutableMap<String, User> = mutableMapOf()
+    private val songMap: MutableMap<String,Song> = mutableMapOf()
+    private val playlistMap: MutableMap<String,Playlist> = mutableMapOf()
+
+
+    override fun getUser(userId: String): Task<User> {
+        return Tasks.forResult(userMap[userId])
+    }
+
+    override fun setUser(userId: String, user: User): Task<Void> {
+        userMap[userId] = User(user.name)
+        return Tasks.forResult(null)
+    }
+
+    override fun getSong(songName: String): Task<Song> {
+        return Tasks.forResult(songMap[songName])
+    }
+
+    override fun registerSong(song: Song): Task<Void> {
+        songMap[song.name] = song
+        return Tasks.forResult(null)
+    }
+
+    override fun getPlaylist(playlistName: String): Task<Playlist> {
+        return Tasks.forResult(playlistMap[playlistName])
+    }
+
+    override fun registerPlaylist(playlist: Playlist): Task<Void> {
+        val titleList : MutableList<Song> = mutableListOf()
+        for( song in playlist.getSongs()){
+            titleList.add(Song(song.name,song.artist,""))
+        }
+        playlistMap[playlist.name] = Playlist(playlist.name,titleList.toSet(),playlist.genre)
+
+        return Tasks.forResult(null)
+    }
+}
```

Comment:
Why does this contain exactly the same code as the LocalDatabase class in our directory with the rest of our backend source code? This seems to cause problems with regard to code coverage, since this "mock implementation is fully tested, but the one in our main directory is not covered at all..

Reply:
I just deleted this one and the other one is in a separate package for better organisation of the project structure